### PR TITLE
Release 0.2.0

### PR DESCRIPTION
**Pull request by JinmingHu-MSFT**
Also add changelog.

### CODE REVIEW

File: Changelog.txt [hunk 1]
```diff
@@ -0,0 +1,16 @@
+Changes in v0.2:
+- Official support for Windows and macOS
+- Compatibility with Visual Studio 2019
+- Support for OAuth authentication
+- Support for Azure Data Lake Storage Gen2 (ADLS Gen2)
+- API names changed: get_blob_property/get_container_property -> get_blob_properties/get_container_properties
+- Add two APIs: set_container_metadata and set_blob_metadata
+- Ramp up API version to 2018-11-09
```
Comment (katmsft) [inline Changelog.txt]: 'Bump up' instead of 'ramp up'?

**Comment by JinmingHu-MSFT** [comment]
@katmsft Can you take time to review the last two commits?

**Comment by JinmingHu-MSFT** [comment]
@katmsft Can you take time to review the last two commits?

**Pull request merged**